checkstyle/checkstyle.xml

```diff
@@ -42,12 +42,6 @@
         <property name="file" value="${main.basedir}/checkstyle/suppressions.xml"/>
     </module>
 
-    <!-- Enable suppression of file contents using //CHECKSTYLE:[ON|OFF] comments -->
-    <module name="SuppressionCommentFilter"/>
```

Comment:
Some of these modules are deprecated but some of them are just replaced. For instance, this one goes under TreeWalker module. So, instead of removing them could we consider to put under right modules? I believe this is where we should look at: https://checkstyle.sourceforge.io/releasenotes.html

Reply:
Good point. Keep in mind that I removed some on purpose (like Cyclomatic Complexity/NPath) because we'd need to either set it to an abnormally high level or just exclude manually since we're not writing a _typical_ code here